values-schema.yaml

```diff
@@ -207,6 +214,61 @@ definitions:
       tag:
         type: string
     title: Container image
+  jobs:
+    description: You may pass an array of Job objects. The Kubernetes API accepts
+      a Job specification, which is meant for run-to-completion apps (e.g. batch
+      Jobs).
+    additionalProperties: false
+    items:
+      additionalProperties: false
+      properties:
+        deployAlways:
+          description: |
+            May give a boolean value. This is false by default.
```

Comment:
` May give a boolean value.` - unnecessary as there is a type already indicating it
`This is false by default.` - just indicate it as a property: `default: false`

Reply:
I don't see any response or code change. I do not understand why it is marked as resolved